frontend/src/leaderboard.js

```diff
@@ -1,36 +1,87 @@
-const ACCOUNTS_URL = 'http://localhost:3000/accounts';
 
-document.getElementById('leaderboard-button').addEventListener('click', () => {
-  const leaderboard = document.getElementById('leaderboard-window');
-  while (leaderboard.firstChild) {
-    leaderboard.removeChild(leaderboard.firstChild);
+class Leaderboard {
+  constructor() {
+    this.leaderboardDiv = document.getElementById('users-container');
+    this.leaderboardWindow = document.getElementById('leaderboard-window');
+    this.leaderboardToggleButton = document.getElementById('leaderboard-toggle-button');
```

Comment:
reminder to self: you changed the name to leaderboard-toggle-button. I need to update it in the wordInteractions.js file to match

Reply:
This is actually the change that broke the play game working. Make sure to test that your name updates don't break any functionality! This change no longer requires players to sign in, and there is now a javascript error when hitting play since `    document.getElementById('leaderboard-button').classList.add('hidden');` no longer exists (attempts to access class list of null object)

Reply:
my bad! updated!